### ISSUE TITLE
admob-plus-cordova-native - cannot find viewProvider:

### ISSUE THREAD

**Issue by vandilsondamata** [admob-plus/admob-plus#662]
**Describe the bug**
I'm facing this error "cannot find viewProvider: " when trying to display native ads.
admob-plus-cordova-native it's installed.

**Expected behavior**
The ad display

**Screenshots**
![Print](https://github.com/admob-plus/admob-plus/assets/85501819/472a09b0-0cbb-4617-9175-9bf40c29badc)

**Environment**

 AdMob Plus Environment Info:
    System:
      OS: Windows 10 10.0.19044
      CPU: (4) x64 Intel(R) Core(TM) i5-2400S CPU @ 2.50GHz
      Memory: 5.01 GB / 15.89 GB
    Binaries:
      Node: 18.16.0 - C:\Program Files\nodejs\node.EXE
      npm: 9.5.1 - C:\Program Files\nodejs\npm.CMD
    IDEs:
      Android Studio: AI-222.4459.24.2221.10121639
    Languages:
      Java: 11.0.17
    npmPackages:
      admob-plus-cordova: 2.0.0-alpha.16 => 2.0.0-alpha.16
      admob-plus-cordova-native: ^1.0.0-alpha.17 => 1.0.0-alpha.17
      cordova-android: ^12.0.1 => 12.0.1
      cordova-browser: 6.0.0 => 6.0.0
      cordova-ios: 6.2.0 => 6.2.0
    Cordova:
      Version: 12.0.0 (cordova-lib@12.0.1)
      Plugins:
        admob-plus-cordova: 2.0.0-alpha.16
        admob-plus-cordova-native: ^1.0.0-alpha.17
    Ionic:
      Version: -
    AndroidManifest.xml:
      Version: -

**Additional context**

showNativeAd() {
    const ad = new admob.NativeAd({
      adUnitId: 'ca-app-pub-3940256099942544/3986624511',
    })

    return ad
      .load()
      .then(() =>
        ad.show({
          x: 0,
          y: 30,
          width: window.screen.width,
          height: 300,
        }),
      )
      .then(
        () =>
          new Promise((resolve) =>
            setTimeout(() => {
              ad.hide()
              resolve()
            }, 5000),
          ),
      )
      .then(() => ad.showWith(document.getElementById('native-ad')))
  }

**Comment by ratson** [comment]
@vandilsondamata Let me know if the latest version fix this.

**Comment by vandilsondamata** [comment]
I just tested, the problem persists.
admob-plus-cordova-native@1.0.0-alpha.18

**Comment by ratson** [comment]
@vandilsondamata Would you check `npx admob-plus doctor` and provide `npx admob-plus info`?

**Comment by vandilsondamata** [comment]
npx admob-plus doctor

√ googleads.g.doubleclick.net
‼ package.json
  × cordova.plugins.admob-plus-cordova.APP_ID_ANDROID
    › Test ID is used: ca-app-pub-3940256099942544~3347511713
√ Cordova Android dependencies
√ config.xml
↓ platforms/ios/*.xcodeproj
√ plugins/admob-plus-cordova/package.json

npx admob-plus info

 AdMob Plus Environment Info:
    System:
      OS: Windows 10 10.0.19044
      CPU: (4) x64 Intel(R) Core(TM) i5-2400S CPU @ 2.50GHz
      Memory: 3.05 GB / 15.89 GB
    Binaries:
      Node: 18.16.0 - C:\Program Files\nodejs\node.EXE
      npm: 9.5.1 - C:\Program Files\nodejs\npm.CMD
    IDEs:
      Android Studio: AI-222.4459.24.2221.10121639
    Languages:
      Java: 11.0.17
    npmPackages:
      admob-plus-cordova: 2.0.0-alpha.16 => 2.0.0-alpha.16
      admob-plus-cordova-native: 1.0.0-alpha.18 => 1.0.0-alpha.18
      cordova-android: ^12.0.1 => 12.0.1
      cordova-browser: 6.0.0 => 6.0.0
      cordova-ios: 6.2.0 => 6.2.0
    Cordova:
      Version: 12.0.0 (cordova-lib@12.0.1)
      Plugins:
        admob-plus-cordova: 2.0.0-alpha.16
        admob-plus-cordova-native: 1.0.0-alpha.18
    Ionic:
      Version: -
    AndroidManifest.xml:
      Version: -

**Comment by ratson** [comment]
Please update `admob-plus-cordova` to `2.0.0-alpha.17` and try again.

**Comment by vandilsondamata** [comment]
Now the error message is gone, but nothing happens. The native ad is not displayed.

![print2](https://github.com/admob-plus/admob-plus/assets/85501819/b6f18cc6-bab2-48f7-89c6-cdf750ffb4f5)

npx admob-plus info

  AdMob Plus Environment Info:
    System:
      OS: Windows 10 10.0.19044
      CPU: (4) x64 Intel(R) Core(TM) i5-2400S CPU @ 2.50GHz
      Memory: 621.30 MB / 15.89 GB
    Binaries:
      Node: 18.16.0 - C:\Program Files\nodejs\node.EXE
      npm: 9.5.1 - C:\Program Files\nodejs\npm.CMD
    IDEs:
      Android Studio: AI-222.4459.24.2221.10121639
    Languages:
      Java: 11.0.17
    npmPackages:
      admob-plus-cordova: 2.0.0-alpha.17 => 2.0.0-alpha.17
      admob-plus-cordova-native: 1.0.0-alpha.18 => 1.0.0-alpha.18
      cordova-android: ^12.0.1 => 12.0.1
      cordova-browser: 6.0.0 => 6.0.0
      cordova-ios: 6.2.0 => 6.2.0
    Cordova:
      Version: 12.0.0 (cordova-lib@12.0.1)
      Plugins:
        admob-plus-cordova: 2.0.0-alpha.17
        admob-plus-cordova-native: 1.0.0-alpha.18
    Ionic:
      Version: -
    AndroidManifest.xml:
      Version: -

npx admob-plus doctor

√ googleads.g.doubleclick.net
‼ package.json
  × cordova.plugins.admob-plus-cordova.APP_ID_ANDROID
    › Test ID is used: ca-app-pub-3940256099942544~3347511713
√ Cordova Android dependencies
√ config.xml
↓ platforms/ios/*.xcodeproj
√ plugins/admob-plus-cordova/package.json

**Comment by ratson** [comment]
@vandilsondamata Are you using `ca-app-pub-3940256099942544/2247696110` for the `adUnitId`?
You may refer to [the updated example code](https://github.com/admob-plus/admob-plus/blob/master/examples/cordova/www/js/index.js#L181).

**Comment by vandilsondamata** [comment]
Yes, I updated the example code with the new adUnitId. Even so, it still doesn't display.
(Android Studio Log Print)
![Print Log](https://github.com/admob-plus/admob-plus/assets/85501819/40e4ccef-08bf-45a2-be14-9703d5c125f3)

![Print 4](https://github.com/admob-plus/admob-plus/assets/85501819/bb310d6d-cb65-4f2d-af07-8c2201f4c6c6)

admob-plus doctor
√ googleads.g.doubleclick.net
‼ package.json
  × cordova.plugins.admob-plus-cordova.APP_ID_ANDROID
    › Test ID is used: ca-app-pub-3940256099942544~3347511713
√ Cordova Android dependencies
√ config.xml
↓ platforms/ios/*.xcodeproj
√ plugins/admob-plus-cordova/package.json

admob-plus info

  AdMob Plus Environment Info:
    System:
      OS: Windows 10 10.0.19044
      CPU: (4) x64 Intel(R) Core(TM) i5-2400S CPU @ 2.50GHz
      Memory: 3.39 GB / 15.89 GB
    Binaries:
      Node: 18.16.0 - C:\Program Files\nodejs\node.EXE
      npm: 9.5.1 - C:\Program Files\nodejs\npm.CMD
    IDEs:
      Android Studio: AI-222.4459.24.2221.10121639
    Languages:
      Java: 11.0.17
    npmPackages:
      admob-plus-cordova: 2.0.0-alpha.17 => 2.0.0-alpha.17
      admob-plus-cordova-native: 1.0.0-alpha.18 => 1.0.0-alpha.18
      cordova-android: ^12.0.1 => 12.0.1
      cordova-browser: 6.0.0 => 6.0.0
      cordova-ios: 6.2.0 => 6.2.0
    Cordova:
      Version: 12.0.0 (cordova-lib@12.0.1)
      Plugins:
        admob-plus-cordova: 2.0.0-alpha.17
        admob-plus-cordova-native: 1.0.0-alpha.18
    Ionic:
      Version: -
    AndroidManifest.xml:
      Version: -

**Comment by ratson** [comment]
@vandilsondamata Would you create a test repo for me to check?
I have no problem running the example code in this repo.

**Comment by vandilsondamata** [comment]
After the corrections you made, I was able to view the native ads on a real device (not on the emulator). Thank you very much for the support!
